docs/code-reviews/inclusion-in-code-review.md

```diff
@@ -46,6 +46,26 @@ Below are some points which emphasize why inclusivity in code reviews is importa
 We as CSE, may come across situations in which code reviews are not ideal and often we are observing non inclusive code review behaviors. Its important to be aware of the fact that culture and communication style of a particular geography also influences how people interact over pull requests.
 In such cases, assuming positive intent of the author and reviewer is a good start to start analyzing quality of code reviews.
 
+## Dealing with the Impostor Phenomenon
+
+Impostor phenomenon is a psychological pattern in which an individual doubts their skills, talents, or accomplishments and has a persistent internalized fear of being exposed as a "fraud" - [wikipedia](https://en.wikipedia.org/wiki/Impostor_phenomenon).
+
+Someone experiencing impostor phenomenon may find submitting a code review particularly stressful. It is important to realise that everybody can have meaningful contributions and to not let the perceived weaknesses prevent contributions.

```

Comment:
```suggestion
Someone experiencing impostor phenomenon may find submitting code for a review particularly stressful. It is important to realize that everybody can have meaningful contributions and not to let the perceived weaknesses prevent contributions.
```

Reply:
Since the Wikipedia quote in the beginning of the section is in American English, I'd have it like so for the rest of the section too. Hence, the "nit".